test/functional/rpc_blockchain.py

```diff
@@ -395,17 +395,55 @@ def _test_getblock(self):
         miniwallet.send_self_transfer(fee_rate=fee_per_kb, from_node=node)
         blockhash = node.generate(1)[0]
 
+        def check_fee_not_in_block(verbosity):
+            block = node.getblock(blockhash, verbosity)
+            assert 'fee' not in block['tx'][1]
+
+        def check_fee_in_block(verbosity):
+            block = node.getblock(blockhash, verbosity)
+            tx = block['tx'][1]
+            assert 'fee' in tx
+            assert_equal(tx['fee'], tx['vsize'] * fee_per_byte)
+
+        def check_vin_contains_prevout(verbosity):
```

Comment:
The parameter is always `3` so it seems unnecessary.

The function does more than checking that prevouts are present. Maybe `{verify|assert}_level3_block`?

Reply:
I renamed the prefix to assert. The pattern i was going for was: `assert_<some-expected-behaviour>(verbosity_level)`, which is consistent with the other functions. I think i'd prefer leaving it as such.

Reply:
ok